wapi/events.py

```diff
@@ -107,7 +107,7 @@ def __init__(self, sse_event):
         self._raw_event = sse_event
         try:
             self.json_data = json.loads(sse_event.data)
-        except:
+        except Exception:
```

Comment:
same as above

Reply:
This is exactly why CodeQL was suggestion. Just writing `except` means you handle `BaseException` which  has three subclass - 1. `Exception` - Which is the base class for all other exceptions; 2. `SystemExit` and 3. `KeyboardInterrupt`

In this case, after `except`, it only `pass`es. Probably we dont want to do this in case of 2 and 3